app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntryEffectHandlerTest.kt

```diff
@@ -4,26 +4,37 @@ import com.nhaarman.mockitokotlin2.mock
 import com.nhaarman.mockitokotlin2.verify
 import org.junit.After
 import org.junit.Test
+import org.simple.clinic.TestData
 import org.simple.clinic.mobius.EffectHandlerTestCase
 import org.simple.clinic.util.TestUserClock
-import org.simple.clinic.util.scheduler.TrampolineSchedulersProvider
+import org.simple.clinic.util.scheduler.TestSchedulersProvider
 import java.time.LocalDate
 import java.time.ZoneOffset.UTC
+import java.util.UUID
 
 class BloodPressureEntryEffectHandlerTest {
+
   private val ui = mock<BloodPressureEntryUi>()
   private val userClock = TestUserClock()
+
+  private val facility = TestData.facility(uuid = UUID.fromString("7a7df523-8397-4c9e-bcef-d0047ea2e969"))
+  private val user = TestData.loggedInUser(
+      uuid = UUID.fromString("5f8c9705-6732-4d1c-aea3-3b5ab10d4a0e"),
+      registrationFacilityUuid = facility.uuid,
+      currentFacilityUuid = facility.uuid
+  )
+
   private val effectHandler = BloodPressureEntryEffectHandler(
       ui = ui,
-      userSession = mock(),
-      facilityRepository = mock(),
       patientRepository = mock(),
       bloodPressureRepository = mock(),
       appointmentsRepository = mock(),
       userClock = userClock,
-      schedulersProvider = TrampolineSchedulersProvider(),
-      uuidGenerator = mock()
-  ).build()
+      schedulersProvider = TestSchedulersProvider.trampoline(),
+      uuidGenerator = mock(),
+      currentUser = { user }
+  ) { facility }.build()
```

Comment:
Can we move this inside the constructor, please? (or is there a reason it's trailing lambda?)

Reply:
Huh, I thought I had corrected this. Maybe I missed it. Will fix.